actions/actions.py

```diff
@@ -23,18 +74,19 @@ def validate_alter(
         tracker: Tracker,
         domain: DomainDict
     ) -> Dict[Text, Any]:
-        try: 
+        try:
```

Comment:
Die Änderungen hier in der actions.py sollte evlt. @noopliez  überprüfen, welche der Versionen für die jeweiligen ValidateInformation Punkte die aktuelle und funktionierende ist. Er kennt sich da am besten aus

Reply:
Mach ich, wenn ich die Merge Conflicts auflöse